examples/solids/index.rst

```diff
@@ -322,27 +327,27 @@ It is crucial to distinguish between the current and reference element in the To
 
     \int_{\Omega}{\boldsymbol{v} \cdot \left(\nabla_X \cdot \boldsymbol{P} + J \rho \boldsymbol{g}\right)} dV = \boldsymbol{0}
 
-By Integration by part we arrive at the weak form:
+Integrating by parts, we arrive at the weak form:
 
 .. math::
 
     \int_{\Omega}{\nabla_X \boldsymbol{v} \colon \boldsymbol{P}}dV =
   - \int_{\Omega}{\boldsymbol{v} \cdot \rho_0 \boldsymbol{g}}dV
   - \int_{\partial \Omega}{\boldsymbol{v} \cdot \boldsymbol{P} \cdot \hat{\boldsymbol{N}}}dA
     
-where :math:`\boldsymbol{P} \cdot \hat{\boldsymbol{N}}` is a prescribed boundary written in terms of reference
-configuration.
+where :math:`\boldsymbol{P} \cdot \hat{\boldsymbol{N}}` is a prescribed boundary
+condition written in terms of the reference configuration.
 
-The constitutive law in indicial notation is given by:
+For simplicity, we express here the constitutive law in indicial notation:
 
 .. math::
 
-   P_{iI} = F_{iB}S_{BI} 
+   P_{iI} = F_{iB}S_{BI}\, ,
 
-Therfore, its material derivative is given by
+so that we can define its Jacobian (or stiffness) fourth order tensor as
```

Comment:
Hmm, I find this confusing terminology here. The analog of the linear elasticity tensor would be `\mathsf C = dS/dE`, e.g., as described here by Wriggers (the irritating factors of 2 evaporate if you use Green-Lagrange strain E instead of right Cauchy-Green tensor C).
![Screenshot_20200320_174203](https://user-images.githubusercontent.com/3303/77213891-1d91eb00-6ad2-11ea-881a-b0bb187fc4f0.png)

Reply:
I agree the "(stiffness)" part could be confusing, in relation to what you called stiffness tensor above. I used the Hughes classical terminology of "material moduli matrix" or "elasticity matrix of moduli" in the past, for which this now would lead to no confusion regarding the stiffness (if this is what you meant). We can also use the incremental terminology as Wriggers does. (I have used algorithmic, too, when expressing discrete stress/strain relationships). But in any case, the existing terminology was not adequate in my opinion.

Reply:
I agree that rewording is needed. I'd prefer consistency with the linear terminology, so call this the incremental elasticity tensor or whatever. We can leave this for @ArashMehraban to think about.